bridge/client/app/_components/ktb-secrets-list/ktb-secrets-list.component.html

```diff
@@ -0,0 +1,60 @@
+<div fxFlexFill>
+  <div class="container" fxFlex fxLayout="column">
+    <div class="mb-3" fxFlex fxLayout="column">
+      <h2>Secrets</h2>
+      <p class="mt-0 mb-3">
+        Secrets are used to store e.g. credentials for the Keptn API or other integrations.
+      </p>
+
+      <dt-table [dataSource]="tableEntries">
+        <ng-container dtColumnDef="secretname" dtColumnAlign="text" dtColumnProportion="8">
+          <dt-header-cell *dtHeaderCellDef>Name</dt-header-cell>
+          <dt-cell *dtCellDef="let row" [textContent]="row.name || 'n/a'"></dt-cell>
+        </ng-container>
+
+        <ng-container dtColumnDef="action" dtColumnAlign="right" dtColumnProportion="1">
+          <dt-header-cell *dtHeaderCellDef></dt-header-cell>
+          <dt-cell *dtCellDef="let row">
+            <button dt-icon-button variant="nested" aria-label="An example nested button containing an icon" (click)="triggerDeleteSecret(row)">
+              <dt-icon name="delete"></dt-icon>
+            </button>
+          </dt-cell>
+        </ng-container>
+
+        <dt-header-row
+          *dtHeaderRowDef="['secretname', 'action']"
+        ></dt-header-row>
+        <dt-row
+          *dtRowDef="
+            let row;
+            columns: ['secretname', 'action']
+          "
+          class="pointer"
+        ></dt-row>
+      </dt-table>
+
+      <div fxLayout="row" fxLayoutAlign="none end" fxLayoutGap="15px" class="mt-3">
+        <button dt-button routerLink="add">
+          <dt-icon name="plus-add"></dt-icon>
+          Add secret
+        </button>
+      </div>
+    </div>
+  </div>
+</div>
+
+<dt-confirmation-dialog [state]="deleteSecretDialogState" aria-label="Delete secret confirmation">
```

Comment:
If a secret is deleted and right after I click on delete of another secret, the dialog `Are you sure you want to delete` disappears because the timeout for `Secret deleted successfully!` ran out.